dataset-registry/src/main/scala/org/sunbird/obsrv/model/ES_datamodels

```diff
@@ -0,0 +1,49 @@
+import org.elasticsearch.common.xcontent.XContentType
+import org.elasticsearch.action.index.IndexRequest
+import org.elasticsearch.action.index.IndexResponse
+import com.fasterxml.jackson.databind.ObjectMapper
```

Comment:
did you get a chance to take a look at the Kafka connector? https://github.com/confluentinc/kafka-connect-elasticsearch